src/constants/index.js

```diff
@@ -29,3 +29,27 @@ export const values2 = [
 ];
 
 export const currencyName = 'USD';
+
+export const englishInfo = {

```

Comment:
This must function as a service

Reply:
It's temporary until the backend team develops the endpoint

Reply:
Should I add a comment that explains it?

Reply:
I added the comment